src/OmniSharp.Abstractions/Options/OmniSharpOptions.cs

```diff
@@ -4,6 +4,8 @@ namespace OmniSharp.Options
 {
     public class OmniSharpOptions
     {
+        public CodeActionOptions CodeActions { get; set; } = new CodeActionOptions();
```

Comment:
Why is this settable and instantiated here? Shouldn't it follow the same pattern as FormattingOptions below?